neet/interfaces.py

```diff
@@ -89,7 +91,7 @@ def neighbors(net,index=None,direction='both',**kwargs):
     eg. ``if size(eca)==3 and boundary!=None:``
     The organization of the neighbors list is as follows:
     ``[node_0|node_1|node_2|left_boundary|right_boundary]``
```

Comment:
I think I have this backwards in the docstring, and it should say `[node_0|node_1|node_2|right_boundary|left_boundary]` but we should double check the code

Reply:
Yeah, the code made it in the correct order.